egs/babel/s5d/local/lattice_to_ctm.sh

```diff
@@ -43,10 +44,25 @@ for f in $lang/words.txt $model $data/segments $data/reco2file_and_channel $dir/
   [ ! -f $f ] && echo "$0: expecting file $f to exist" && exit 1;
 done
 
+if [ -f $dir/../frame_shift ]; then
```

Comment:
I believe I copied this from switchboard. https://github.com/kaldi-asr/kaldi/blob/master/egs/swbd/s5c/local/score_sclite.sh